UKHomeOffice/sgmr-service
# NMSW-763 Remove instances of ais

### AC / Description of changes

remove AIS reformatting from SubmitPanel.jsx page

### To Test

- Check that the code does not contain instances if AIS

Can also check by:
- Run app
- Sign in
- Create a voyage
- Submit the voyage
- Find the voyage you just submitted and change something
- Click save and continue
- > You should be returned to the CYA page and be able to submit successfully

### Notes

---

**Remember to merge to feature branches not `master`.**
